docs/how-to-use-clusterctl.md

```diff
@@ -0,0 +1,45 @@
+## Using `clusterctl` to create a cluster from scratch
+
+This document provides an overview of how `clusterctl` works and explains how one can use `clusterctl` to create a kubernetes cluster from scratch.
+
+### What is `clusterctl`?
+`clusterctl` is a CLI tool to create a kubernetes cluster. It uses Cluster API provider implementations to provision resourses needed by the kubernetes cluster.
```

Comment:
There should probably be a statement here to let users know that `clusterctl` is provided by provider implementations and not a single common tool.

Reply:
Good point. I will add a statement to make it clear.

Reply:
Typo: resourses should be resources.